dwi_ml/data/hdf5_creation.py

```diff
@@ -0,0 +1,242 @@
+                              '{}. Wrong dimensions?'.format(data_file, group))
+
+    # Save unnormalized data
+    if save_intermediate:
+        logging.debug('  Saving intermediate non standardized files.')
+        group_image = nib.Nifti1Image(group_data, group_affine)
+        output_fname = "{}_nonstandardized.nii.gz".format(group)
+        nib.save(group_image,
+                 str(subj_output_path.joinpath(output_fname)))
+
+    # Standardize data
+    logging.debug('  Standardizing data')
+    standardized_group_data = standardize_data(group_data, subj_mask_data)
+
+    # Save standardized data
+    if save_intermediate:
+        logging.debug('  Saving intermediate standardized files.')
+        standardized_img = nib.Nifti1Image(standardized_group_data,
+                                           group_affine)
+        output_fname = "{}_standardized.nii.gz".format(group)
+        nib.save(standardized_img,
+                 str(subj_output_path.joinpath(output_fname)))
+
+    return standardized_group_data, group_affine, group_header
+
+
+def process_streamlines(bundles_dir: Path, bundles,
+                        header: nib.Nifti1Header, step_size: float,
+                        space: Space):
+    """Load and process a group of bundles and merge all streamlines
+    together.
+
+    Parameters
+    ----------
+    bundles_dir : Path
+        Path to bundles folder.
+    bundles: List[str]
+        List of the bundles filenames to load.
+    header : nib.Nifti1Header
+        Reference used to load and send the streamlines in voxel space and to
+        create final merged SFT.
+    step_size: float
+        Step size to resample streamlines. If none, compress streamlines.
+    space: Space
+        Space to place the tractograms.
+
+    Returns
+    -------
+    output_tractogram : StatefulTractogram
+        All streamlines in voxel space.
+    output_lengths : List[float]
+        The euclidean length of each streamline
+    """
+    # Silencing SFT's logger if our logging is in DEBUG mode
+    set_sft_logger_level('WARNING')
+
+    # Initialize
+    output_tractogram = None
+    output_lengths = []
+
+    # Find official bundles list
+    if bundles is None:
+        # Take everything found in subject bundle folder
+        bundles = [str(p) for p in bundles_dir.glob('*')]
+        if len(bundles) == 0:
+            raise ValueError("No bundle found in the bundles folder!")
+
+    for bundle_name in bundles:
+        # Find bundle name
+        # Note. glob uses str. Other possibility: Path.cwd().glob but creates
+        # a generator object.
+        logging.debug('    Loading bundle {}'.format(bundle_name))
+        bundle_possible_name = str(bundles_dir.joinpath('*' +
+                                                        bundle_name + '*'))
+        bundle_real_name = glob.glob(bundle_possible_name)
+        if len(bundle_real_name) == 0:
+            raise ValueError('Bundle {} not found!'.format(bundle_name))
```

Comment:
I don't get this check and the one below. What's the situation where these errors could be triggered ?

Reply:
Yeah, I think I will remove that. 
At the time, I thought it was bad that your bundle list couldn't be, ex: OR_L (because recobundle's output is OR_L_m). I wanted to find automatically all bundles that have more or less the name you want. But now I think it's better not.

Reply:
Do we allow a bundle to be not found, however? Should it be an option?

Reply:
Missing bundles will definitely be a thing, we should support it